app/components/SearchBox.tsx

```diff
@@ -34,24 +34,23 @@ const SearchBox: React.FunctionComponent<Props> = ({
     type: 'object',
     properties: {
       searchField: {
+        title: 'Search By:',
         type: 'string',
         enum: dropdownSortItems
       },
       searchValue: {
+        title: 'Search Term:',
```

Comment:
Sets `<label>`s for SearchBox's "search by" dropdown and search term `<input/>`s 
(rule: all inputs must have a corresponding label)